src/query/block/series.go

```diff
@@ -52,24 +54,38 @@ func (s Series) Len() int {
 
 // UnconsolidatedSeries is the series with raw datapoints.
 type UnconsolidatedSeries struct {
-	datapoints ts.Datapoints
-	Meta       SeriesMeta
+	opts UnconsolidatedSeriesOptions
```

Comment:
This is a little misleading with the series data itself inside an "option" struct now right? Seems like it might be better to have just a new 
```
type UnconsolidatedSeriesStats struct {
    Enabled         bool	
    DecodeTime time.Duration
}
```
within the original `UnconsolidatedSeries` struct